### PR TITLE
⬆️ Alpha 12

### PR DESCRIPTION
**Pull request by hzoo**
> (node:79847) DeprecationWarning: Passing lineNumber and colNumber is deprecated to babel-code-frame. Please use `codeFrameColumns` 😅

### CODE REVIEW

File: packages/babel-core/src/transformation/file/index.js [hunk 1]
```diff
@@ -432,7 +432,13 @@ export default class File extends Store {
 
       const loc = err.loc;
       if (loc) {
-        err.codeFrame = codeFrame(code, loc.line, loc.column + 1, this.opts);
+        const location = {
+          start: {
+            line: loc.line,
+            column: loc.column + 1,
```
Comment (SimenB) [inline packages/babel-core/src/transformation/file/index.js]: This sucks... Is the `+1` expected?

Comment (hzoo) [inline packages/babel-core/src/transformation/file/index.js]: Oh? in our code it was +1 already? I just copied it over

Comment (SimenB) [inline packages/babel-core/src/transformation/file/index.js]: I'm thinking it should just work :tm: with existent AST loc objects. That was the point.

Comment (hzoo) [inline packages/babel-core/src/transformation/file/index.js]: We don't have any tests for it probably so I didn't really check, dono if you want to take a look

**Comment by SimenB** [comment]
still calls deprecated method: #5815

**Pull request merged**